Gemfile

```diff
@@ -0,0 +1,27 @@
+source "https://rubygems.org"
+ruby "2.7.4"
+
+gem "jekyll", "~> 4.2.1"
```

Comment:
How did we get this Gemfile?
```
bundle init
```

Jekyll is a Gem! so are so many Jekyll plugins.
You can explore each of them and see what cool functionalities they can add.